templates/csv.yaml

```diff
@@ -444,8 +444,9 @@ spec:
                       runAsNonRoot: true
                       capabilities:
                         drop:
-                          - all
-
+                          - ALL
+                      seccompProfile:
```

Comment:
Afaik this is available since 1.19 so we should not have issues on previous versions.

Reply:
Unfortunately on 4.6 you need special handling: https://docs.openshift.com/container-platform/4.6/security/seccomp-profiles.html